kubernetes/samples/charts/apache-webtier/README.md

```diff
@@ -63,11 +66,14 @@ installing the chart. For example:
 ```console
 $ helm install my-release --values values.yaml apache-webtier
 ```
+## useNonPriviledgedPorts
+By default, the chart will install the Apache webtier on PriviledgedPort i.e port 80. You need to have the flag `useNonPriviledgedPorts=true` to enable the Apache webtier to listen on port `8080`
+
 
 ## RBAC
 By default, the chart will install the recommended RBAC roles and role bindings.
 
-You need to have the flag `--authorization-mode=RBAC` on the API server. See the following document for how to enable [RBAC](https://kubernetes.io/docs/admin/authorization/rbac/).
+You need to have the flag `--authorization-mode=RBAC` on the API server. See the following document for how to enable [RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/).
```

Comment:
You need to have the flag -> Set the flag

Reply:
Updated.